### ISSUE TITLE
Specific Patreon ID formats not being recorded in artist entries

### ISSUE THREAD

**Issue by baconmeh** [danbooru/danbooru#5847]
![image](https://github.com/user-attachments/assets/7758fa07-d946-40af-9e08-1edc995f8c85)
![image](https://github.com/user-attachments/assets/24bc8d5a-c523-4923-a8e2-13e647bdcdd8)
In some cases, a link such as https://patreon.com/jamescabello will turn into https://www.patreon.com/profile/creators?u=7422057. I clicked James' Patreon link "on his Newgrounds profile":[https://danbooru.donmai.us/artists/203352]. On the surface it displays a preview as the /jamescabello link. When clicking it, it reformats into /creators?u=7422057 when you arrive. I expected this to reformat into /user?u=57431512, but it instead reformatted into  https://www.patreon.com/profile. I must presume the same thing happened with those "Duplicate" entries, which bare the same URL.

tl;dr- Fix https://www.patreon.com/profile/creators?u=7422057 links so they reformat as https://www.patreon.com/creators?u=7422057. The /profile seems to be confusing the system.

[EDIT] It's worse than I thought. Fixing the link to read as https://www.patreon.com/creators?u=7422057 does NOT fix the URL reformatting itself into /profile.

**Comment by nonamethanks** [comment]
https://www.patreon.com/creators?u=7422057 is actually wrong. It redirects to https://www.patreon.com/profile/creators?u=5016401 because patreon user 5016401 has the unfortunate username "creators". The canonical url should be /profile/creators.
